cql/src/main/java/org/apache/cassandra/stargate/transport/internal/CqlServer.java

```diff
@@ -21,13 +21,7 @@
 import io.netty.bootstrap.ServerBootstrap;
 import io.netty.buffer.ByteBuf;
 import io.netty.buffer.ByteBufAllocator;
-import io.netty.channel.Channel;
-import io.netty.channel.ChannelFuture;
-import io.netty.channel.ChannelHandlerContext;
-import io.netty.channel.ChannelInitializer;
-import io.netty.channel.ChannelOption;
-import io.netty.channel.ChannelPipeline;
-import io.netty.channel.EventLoopGroup;
+import io.netty.channel.*;
```

Comment:
Seems that IntelliJ is collapsing the imports.. We agreed at some point to not have the collapsed imports.. Can you increase the number of imports to use `*` to something very high (it's in Editor > Code Style > Java, then Imports tab)..

Reply:
I was wondering about that; I think many IDEs default to collapse at 5 or so. I'll change my settings.

Reply:
Reverted import change as suggested.